Atomic/syscontainers.py

```diff
@@ -398,12 +434,18 @@ def _get_ostree_repo(self):
 
     def update_system_container(self, name):
         self.args.display = False
+        self.args.remote = None
 
         repo = self._get_ostree_repo()
         if not repo:
             raise ValueError("Cannot find a configured OSTree repo")
 
         path = os.path.join(self._get_system_checkout_path(), name)
+        with open(os.path.join(path, "info"), 'r') as info_file:
+            info = json.loads(info_file.read())
+            self.args.remote = info['remote']
+            if self.args.remote:
+                util.write_out("Updating a container with a remote rootfs. Only changes to config will be applied.")
```

Comment:
if we use the original path used for the installation and it resolves to a different location, i.e. the symlink points somewhere else, then the container is also updated.  Should we remove this warning?